sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -198,39 +238,36 @@ Upload a block blob.
 ```C# Snippet:SimpleBlobUpload
 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
     sourceResource: files.FromFile(sourceLocalPath),
-    destinationResource: blobs.FromClient(destinationBlob));
+    destinationResource: blobs.FromBlob(destinationBlobUri));
 await dataTransfer.WaitForCompletionAsync();
 ```
 
 Upload a directory as a specific blob type.
 
 ```C# Snippet:SimpleDirectoryUpload
 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
-    sourceResource: files.FromDirectory(sourcePath),
-    destinationResource: blobs.FromClient(
-        blobContainerClient,
+    sourceResource: files.FromDirectory(sourceLocalPath),
+    destinationResource: blobs.FromContainer(
+        blobContainerUri,
         new BlobStorageResourceContainerOptions()
         {
             // Block blobs are the default if not specified
             BlobType = BlobType.Block,
             BlobDirectoryPrefix = optionalDestinationPrefix,
-        }),
-    transferOptions: options);
+        }));
 await dataTransfer.WaitForCompletionAsync();
 ```
 
 ### Download
 
 A download takes place between a blob `StorageResource` as source and local file `StorageResource` as destination.
 
-Download a block blob.
+Download a blob.
 
 ```C# Snippet:SimpleBlockBlobDownload
-BlobsStorageResourceProvider blobs = new();
-LocalFilesStorageResourceProvider files = new();
 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
-    sourceResource: blobs.FromClient(sourceBlobClient),
-    destinationResource: files.FromFile(downloadPath));
+    sourceResource: blobs.FromBlob(sourceBlobUri),
+    destinationResource: files.FromFile(destinationLocalPath));
 await dataTransfer.WaitForCompletionAsync();
 ```
```

Comment:
Do we want to change the container download sample below to use `FromContainer` or leave as `FromClient` to showcase that path too?